Packs/ArcusTeam/Integrations/ArcusTeam/ArcusTeam.py

```diff
@@ -0,0 +1,187 @@
+            return issue
+    return editIssue
+
+
+def arcusteam_get_devices(client: Client, args: Dict[str, Any]):
+    """
+    Search for matching devices giving a device name
+    :param device_name: device name to search for in the DB.
+    :return: List of matching devices for the given device.
+    """
+    url = urljoin(client._base_url, "/get_devices")
+    payload = {
+        "vendor": args.get("vendor", ""),
+        "model": args.get("model", ""),
+        "series": args.get("series", ""),
+        "firmware_version": args.get("firmware_version", ""),
+        "version": VERSION
+    }
+    result = requests.request("POST", url, headers=client._headers, data=json.dumps(payload))
+    resultJson = result.json()
+    markdown = '## Found ' + str(len(resultJson)) + ' devices\n'
+    markdown += "".join(list(map(deviceToMarkdown, resultJson)))
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix="ArcusTeamDevices",
+        outputs_key_field="",
+        outputs={'devices': resultJson},
+    )
+
+
+def arcusteam_get_vulnerabilities(client: Client, args: Dict[str, Any]) -> CommandResults:
+    url = urljoin(client._base_url, "/get_vulnerabilities")
+    returnFields = str(args.get("return_fields")).split(',')
+    payload = {
+        "firmwareId": args.get("firmware_id", ""),
+        "version": VERSION,
+        "deviceId": args.get("device_id", ""),
+        "pageSize": int(args.get("page_size", 10)),
+        "page": int(args.get("page_number", 1)),
+        "sortOrder": args.get("sort_order", "desc"),
+        "sortField": args.get("sort_field", "risk"),
+        "returnFields": str(args.get("return_fields")).split(','),
+
+    }
+    result = requests.request("POST", url, headers=client._headers, data=json.dumps(payload))
+    if len(result.json().get('code', '')) > 0:
+        raise Exception(result.json().get('message'))
+    resultJson = result.json()
+    resultJson['results'] = list(map(getEditIssue(returnFields), resultJson.get("results")))
+    markdown = '## Scan results\n'
+
+    if len(resultJson.get('results')) > 0:
+        markdown += tableToMarkdown(
+            'Number of CVE\'s found: ' + str(resultJson.get("max_items")),
+            resultJson.get('results'),
+            headers=list(resultJson.get('results')[0].keys())
+        )
+    else:
+        markdown += "No results"
+
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix="ArcusTeamVulnerabilities",
+        outputs_key_field="",
+        outputs=result.json(),
+    )
+
+
+""" MAIN FUNCTION """
+
+
+def main() -> None:
+    """
+    Main function, parses params and runs command functions
+    :return:
+    :rtype:
+    """
+
+    api_key = demisto.params().get("ApiKey")
+
+    client_id = demisto.params().get("ClientID")
+
+    # get the service API url
+    base_url = urljoin(demisto.params()["url"], "/api/v10/xsoar")
```

Comment:
Changing to real default value
```suggestion
    base_url = urljoin(demisto.params()["url"], "https://web.prod.arcusteam.com")
```